src/azul/plugins/metadata/hca/transform.py

```diff
@@ -1613,8 +1642,8 @@ def _transform(self) -> Contribution:
                         ),
                         matrices=matrices,
                         contributed_analyses=contributed_analyses,
-                        aggregate_dates=[self._aggregate_date()],
-                        projects=[self._project(self._api_project)])
+                        projects=[self._project(self._api_project)],
+                        dates=[self._date(self.entity_dates(), self.aggregate_dates())])
```

Comment:
I think the lines could be re-ordered here too:
```suggestion
                        dates=[self._date(self.entity_dates(), self.aggregate_dates())]
                        projects=[self._project(self._api_project)])
```